src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -74,7 +74,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     if (!userStore.isUserRegistered(username)) {
-      User user = new User(UUID.randomUUID(), username, Instant.now());
+      // TODO: add password
+      User user = new User(UUID.randomUUID(), username, null, Instant.now());
+//      User user = new User(UUID.randomUUID(), username, Instant.now());
```

Comment:
readability nit: indentation - the // should start just below the start of the previous line

Reply:
Better: you can just remove this line.